spec/controllers/facility_reservations_controller_spec.rb

```diff
@@ -141,6 +141,10 @@
           expect(assigns[:reservation].admin_note).to eq "Testing"
         end
 
+        it "sets created_by" do
```

Comment:
We should probably have a spec for the offline creation as well.

Reply:
Added.